frontend/src/screens/BackupMnemonic.tsx

```diff
@@ -124,7 +125,21 @@ export function BackupMnemonic() {
               </div>
               <span>
                 Your recovery phrase is a set of 12 words that{" "}
-                <b>backs up your wallet</b>
+                <b>backs up your wallet savings balance</b>.
+                {info?.albyAccountConnected && (
+                  <b>
+                    You also need to make sure you do not forget your unlock
+                    password as this will be used to recover funds from
+                    channels.
+                  </b>
+                )}
+                {!info?.albyAccountConnected && (
+                  <b>
+                    Make sure to also backup your data directory as this is
+                    required to recover funds on your channels. You can also
+                    connect your Alby Account for automatic backups.
```

Comment:
automatic encrypted backups
```suggestion
                    connect your Alby Account for automatic encrypted backups (you still need your seed and unlock password to decrypt those)
```

Reply:
we should make a guides page with all the backup details what is needed in what case.